server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/FileWriter.java

```diff
@@ -247,7 +259,18 @@ public long close() throws IOException {
       waitOnNoPending(notifier.numPendingFlushes);
     } finally {
       returnBuffer();
-      channel.close();
+      try {
+        if (channel != null) {
+          channel.close();
+        }
+        if (stream != null) {
+          stream.flush();
+          stream.close();
```

Comment:
will close call flush?

Reply:
DFSOutputStream will flush when the close method is called.